smdebug/rules/action/stop_training_action.py

```diff
@@ -0,0 +1,60 @@
+# Standard Library
+import os
+
+# Third Party
+import boto3
+
+# First Party
+from smdebug.core.logger import get_logger
+
+
+class StopTrainingAction:
+    def __init__(self, rule_name, training_job_prefix):
+        self._training_job_prefix = training_job_prefix
+        env_region_name = os.getenv("AWS_REGION", "us-east-1")
```

Comment:
does the user set the environment variable? alternatively we can get the region via boto3 `boto3.Session().region_name`

Reply:
it is already set in AWS instances  - training job, rule job ,  notebook etc.